reference/configuration/framework.rst

```diff
@@ -69,19 +69,18 @@ Configuration
     * `gc_probability`_
     * `gc_maxlifetime`_
     * `save_path`_
+* `assets`_
+    * `version`_
+    * `version_format`_
+    * `base_urls`_
+    * `packages`_
```

Comment:
I kept it in the same order as here: http://symfony.com/blog/new-in-symfony-2-7-the-new-asset-component
Also the alphabetic order is not consistent in the framework configuration reference.